internal/plugins/ansible/v1/scaffolds/internal/templates/makefile.go

```diff
@@ -48,20 +46,8 @@ func (f *Makefile) SetTemplateDefaults() error {
 	}
 
 	f.TemplateBody = makefileTemplate
-
 	f.IfExistsAction = file.Error
-
-	if f.Image == "" {
-		f.Image = "controller:latest"
-	}
-
-	if f.KustomizeVersion == "" {
-		f.KustomizeVersion = "v3.5.4"
-	}
-
-	if f.AnsibleOperatorVersion == "" {
-		f.AnsibleOperatorVersion = strings.TrimSuffix(version.Version, "+git")
-	}
+	f.KustomizeVersion = constants.KustomizeVersion
```

Comment:
Mandatory values (`Image`, `AnsibleOperatorVersion`) should be checked, like I mentioned in [`dockerfile.go`](https://github.com/operator-framework/operator-sdk/pull/3559/files#r472500377). Ditto elsewhere.

Reply:
done.